lib/tasks/temp.rake

```diff
@@ -1,4 +1,23 @@
 namespace :temp do
+  desc 'Migrate PublicBody notes into Note model'
+  task migrate_public_body_notes: :environment do
+    scope = PublicBody.where.not(notes: nil)
+    count = scope.count
+
+    scope.with_translations.find_each.with_index do |body, index|
+      PublicBody.transaction do
+        body.legacy_note.save
```

Comment:
```suggestion
        body.legacy_note.save!
```

Reply:
Also want to add this to the upgrade notes in the changelog

Reply:
https://github.com/mysociety/alaveteli/issues/7267